doc/source/locale/zh_CN/LC_MESSAGES/development/contributing_codebase.po

```diff
@@ -37,6 +37,9 @@ msgid ""
 "for stylistic errors. Good style is a requirement for submitting code to "
 "Xorbits."
 msgstr ""
+"写好代码不仅仅是关于你写的内容，也与你如何写有关。"
+"在 :ref:`Continuous Integration <contributing.ci>` 测试期间，我们将运行几个工具来检查你的代码样式错误。"
+"好的代码风格是提交代码给Xorbits的要求。"
```

Comment:
**Please ensure that there is a space before and after numbers or English words in your translation.**

This should be applied consistently throughout the codebase, including other parts of this pull request.

```suggestion
"好的代码风格是提交代码给 Xorbits 的要求。"
```